### PR TITLE
refactor logs into sub pkg

### PR DESCRIPTION
**Pull request by resouer**
ref #15634 

A minor fix to refactor util log into its directory.

### CODE REVIEW

**Comment by resouer** [comment]
@k8s-bot e2e test this please

**Comment by resouer** [comment]
@brendandburns Whenever I remove `util/logs.go` , verify generated docs will report `--log-flush-frequency` is removed like this:
```
contrib/completions/bash/kubectl
 @@ -300,7 +300,6 @@ _kubectl_get()
      flags+=("--kubeconfig=")
      flags+=("--log-backtrace-at=")
      flags+=("--log-dir=")
 -    flags+=("--log-flush-frequency=")
```
Really not quite clear of this logic here, any tips?

**Comment by resouer** [comment]
@mikedanese  Could u please take a look at the failing tests? I'm confused why e2e tests fails due to this minor fix.

**Comment by mikedanese** [comment]
Thanks, I'll take a look tomorrow

**Comment by resouer** [comment]
@mikedanese Any progress or tips on your side?

File: docs/admin/kubelet.md [hunk 1]
```diff
@@ -106,7 +106,6 @@ kubelet
       --kubeconfig="/var/lib/kubelet/kubeconfig": Path to a kubeconfig file, specifying how to authenticate to API server (the master location is set by the api-servers flag).
       --kubelet-cgroups="": Optional absolute name of cgroups to create and run the Kubelet in.
       --lock-file="": <Warning: Alpha feature> The path to file for kubelet to use as a lock file.
-      --log-flush-frequency=5s: Maximum number of seconds between log flushes
```
Comment (mikedanese) [inline docs/admin/kubelet.md]: Do you know what is causing this to be removed?

**Comment by mikedanese** [comment]
Sorry, fell of my radar.

File: docs/admin/kubelet.md (hunk 1 above)
Comment (resouer) [inline docs/admin/kubelet.md]: Whenever I removed `util/logs.go` and move it to sub-dir, this flag will be removed by updating generated docs.

I think it has relationship with `https://github.com/resouer/kubernetes/blob/log-dev/pkg/util/log/logs.go#L30`, but not sure of the root cause ...

File: docs/user-guide/kubectl/kubectl_rollout_pause.md [hunk 2]
```diff
@@ -73,8 +73,7 @@ $ kubectl rollout pause deployment/nginx
       --kubeconfig="": Path to the kubeconfig file to use for CLI requests.
       --log-backtrace-at=:0: when logging hits line file:N, emit a stack trace
       --log-dir="": If non-empty, write log files in this directory
-      --log-flush-frequency=5s: Maximum number of seconds between log flushes
-      --logtostderr[=true]: log to standard error instead of files
+      --logtostderr[=false]: log to standard error instead of files
```
Comment (mikedanese) [inline docs/user-guide/kubectl/kubectl_rollout_pause.md]: Don't we want this default true?

**Comment by mikedanese** [comment]
The fact that this changed --help bothers me. A  refactoring should not cause the flags to change.

**Comment by resouer** [comment]
[Update] If move` var logFlushFreq ` into cmd,  `sudo hack/verify-swagger-spec.sh` will fail with ` Timed out waiting for apiserver:  to answer at http://127.0.0.1:8050/healthz; tried 25 waiting 0.5 between each`

**Comment by mikedanese** [comment]
Please get tests to pass if you want to merge this

**Pull request closed without merging**